### DISCUSSION TITLE
SDF nodes for Malt?

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#200] [Q&A] [answered]
I could have sworn I've seen SDF nodes used with malts at some point but I cant find them currently. Does Malt have SDF shader functions and if not, will that happen in the future?

**Comment by pragma37** [comment, marked as answer]
You can use SDFs:
https://github.com/bnpr/Malt/blob/Development/Shader%20Examples/advanced_sdf.mesh.glsl
But the API is not available through nodes.
